app/components/provider_user_notification_preferences_component.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class='govuk-form-group'>
+  <%= form_with model: notification_preferences, url: form_path, method: :put do |f| %>
+    <% ProviderUserNotificationPreferences::NOTIFICATION_PREFERENCES.each do |notification_preference| %>
+      <%= f.govuk_collection_radio_buttons notification_preference,
+        on_off_options,
+        :value,
+        :name,
+        inline: true,
+        legend: { text: t("provider_user_notification_preferences.#{notification_preference}.legend") } %>
+    <% end %>
+
+    <%= f.govuk_submit 'Save settings' %>
```

Comment:
Can you add some padding around this submit button as per the [protoype](https://manage-applications-beta.herokuapp.com/account/notifications), as when in mobile view there isn't any spacing